### ISSUE TITLE
About the test accuracy of cifar 100 on ResNet-18 without cutout

### ISSUE THREAD

**Issue by KseeleK** [uoguelph-mlrg/Cutout#10]
I have tried to run your code on cifar 100 with ResNet-18, but I can't get the same accuracy as you described in your paper. 
My result is less than 70%, but yours is 77.54% 
(with data augmentation, but without cutout. I just clone your code, and didn't do any change on your model or anything else)

So my question is: Is here any training tricks that didn't mention in your paper or code?

Thank you!

**Comment by TDeVries** [comment]
I didn't do any training tricks, aside from standard procedures like learning rate scheduling that are described in the paper. The code I originally used to get those numbers is almost identical to what is in the repo here. The only change is that I made things a bit neater for the release, but all of the actual processes are the same.

**Comment by KseeleK** [comment]
Thank you for your reply! I have tried to run many times,  but still fail to achieve this accuracy. I think it is may because of the Pytorch version or hardware environment? I use Pytorch 1.4.0 and my environment is two titan x GPUs

Here is the record of my training process (.csv file) :

[cifar100_resnet18.zip](https://github.com/uoguelph-mlrg/Cutout/files/4334218/cifar100_resnet18.zip)

**Comment by TDeVries** [comment]
Thanks for uploading your log!

There's something weird going on here. I plotted the results from your log and one of my logs (with data augmentation, no cutout), and there are a couple differences. The most obvious one is that yours doesn't have the sudden jumps which the learning rate scheduling causes. It looks like in PyTorch 1.4.0 they deprecated the old scheduler, so maybe that's causing the issue. You can try replacing `scheduler.step(epoch)` with `scheduler.step()` in train.py. The other noticeable different is that your model appears to be learning much slower than mine, but I have no idea what might be causing that issue. Your model starts out fast but then slows down a lot, so maybe it's also related to the scheduler. I would try changing that and see how it does.

**You log**
![cifar100_no_scheduling](https://user-images.githubusercontent.com/4457314/76705278-415dc700-66b5-11ea-8acc-570891635a11.PNG)

**My log**
![cifar100_w_scheduling](https://user-images.githubusercontent.com/4457314/76705282-4589e480-66b5-11ea-9e69-1a5d8d113e22.png)

**Comment by KseeleK** [comment]
Thanks for your advice! I manage to achieve the accuracy of 78%. I follow your advice by replacing scheduler.step(epoch) with scheduler.step(), and it works! 

And here is my log:
[cifar100_resnet18.zip](https://github.com/uoguelph-mlrg/Cutout/files/4335871/cifar100_resnet18.zip)

**Comment by TDeVries** [comment]
Thanks for the update! I've added some comments to the code in case anyone else runs into the same issue.
